### PR TITLE
added user Reducer and general redux setup

### PR DESCRIPTION
**Pull request by shrushti2000**
Added the redux setup along with user Reducer.
I have not implemented a proper navigation bar. 
Just roughly added the logout and profile links/buttons.

### CODE REVIEW

**Comment by shrushti2000** [comment]
token00
login3
token

File: client/src/redux/user/action.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const setCurrentUser=user=>({
+    type:'SET_CURRENT_USER',
```
Comment (aashimgarg) [inline client/src/redux/user/action.js:2]: add a seperate file for all the types. user/types.js . Just a simple convention

Comment (shrushti2000) [reply client/src/redux/user/action.js:2]: ok sure .

**Review by aashimgarg — APPROVED** [review]

**Comment by aashimgarg** [comment]
Add proper indentation to the code as well @shrushti2000 as it becomes difficult to have a look then

**Comment by shrushti2000** [comment]
Ohk sure @aashimgarg .
Will take care of this from next time onwards.
sorry for the inconvenience.

**Pull request merged**